src/methods/get.js

```diff
@@ -549,3 +552,22 @@ export function GetTemplateObject(realm: Realm, templateLiteral: BabelNodeTempla
   // 15. Return template.
   return template;
 }
+
+export function GetFromArrayWithWidenedNumericProperty(realm: Realm, arr: ArrayValue, P: string | SymbolValue): Value {
+  let type = Value;
+
+  if (typeof P === "string") {
+    // these are safe methods to allow, as they return a new array
+    // so we use the ordinary get for these cases.
+    if (P === "map" || P === "slice" || P === "filter" || P === "concat") {
```

Comment:
I thought we got rid of this whitelist?

Reply:
No, we need them. We need to access the array prototype methods, but only the ones that create new arrays.